README.md

```diff
@@ -2,16 +2,28 @@
 
 [![License](https://img.shields.io/badge/License-Apache-blue.svg)](LICENSE)
 
-## Description
+# Realm Dart Samples
 
-This repo contains Realm Flutter and Realm Dart SDK samples. 
+This repo contains Flutter and Dart samples that demonstate the usage of [Realm Flutter and Realm Dart SDKs](https://www.mongodb.com/docs/realm/sdk/flutter/).
 
-The Realm Flutter and Realm Dart SDK are located at https://github.com/realm/realm-dart
+To build your own app with Realm, use the [realm](https://pub.dev/packages/realm) package for Flutter and the [realm_dart](https://pub.dev/packages/realm_dart) package for Dart applications.
 
+The Realm Flutter and Dart SDK repository is located at https://github.com/realm/realm-dart
 
-## Usage 
-Check the README.md files of the samples for instructions how to run them.
+You may find the [Quick Start - Flutter SDK]( https://www.mongodb.com/docs/realm/sdk/flutter/quick-start/) helpful if you are using Realm for the first time.
+To learn more about using Realm with Atlas App Services, refer to the [Atlas Device Sync](https://www.mongodb.com/docs/realm/sdk/flutter/sync/) and [connecting to Atlas App Services](https://www.mongodb.com/docs/realm/sdk/flutter/app-services/) documentation.
 
+# Samples list
+* [`Realm Dart sample`](https://github.com/realm/realm-dart-samples/tree/main/realm_dart) - Dart console app that demonstrates how to use a local Realm.
+* [`Provider Shopper`](https://github.com/realm/realm-dart-samples/tree/main/provider_shopper) - A Flutter sample app from [Flutter sample repo](https://github.com/flutter/samples/tree/master/provider_shopper) that shows a state management approach with a local Realm.
```

Comment:
```suggestion
* [`Provider Shopper`](https://github.com/realm/realm-dart-samples/tree/main/provider_shopper) - A Flutter sample app from [Flutter sample repo](https://github.com/flutter/samples/tree/master/provider_shopper) modified to use a local Realm instead of an in-memory collection.
```